test/prints.jl

```diff
@@ -20,15 +18,13 @@
     end
 
     @testset "App show" begin
-        io = IOBuffer()
         app = App()
         add_post!(app, "test") do req
             body = req.body
             return body
         end
 
-        show(io, app)
-        iostr = String(take!(io))
+        iostr = sprint(show, app)
```

Comment:
Or even shorter:
```julia
iostr = repr(app)
```

Reply:
I think `str = repr(app)` is even better!

Reply:
Do you want to do the commit to make this change?
Or do you prefer that I do?